release-common-lib/src/main/java/org/reactome/util/general/CollectionUtils.java

```diff
@@ -29,13 +30,34 @@
 	 * Returns a set with the elements in the passed collection or an empty set if the passed collection is
 	 * null.  This method is meant to guard against null values to eliminate the need for conditionals checking
 	 * for null and to allow uses such as iteration in loops directly.
+	 * @param <E> Type of the elements for the passed collection and returned set
 	 * @param collection Collection of type E
 	 * @return Set of type E with the elements in the collection or an empty Set of type E if the collection is null
 	 */
 	public static <E> Set<E> safeSet(Collection<E> collection) {
 		return Optional.ofNullable(collection).map(CollectionUtils::convertToSet).orElse(Collections.emptySet());
 	}
 
+	/**
+	 * Returns a list with all the elements of the passed lists in the order the lists are received (i.e. flattens
+	 * multiple lists into one).
+	 * @param <E> Type of the elements for the passed lists and returned list
+	 * @param lists List objects of type E to be combined into a single list
+	 * @return Combined list of type E or an empty List of type E if the lists received are null or empty
+	 */
+	@SafeVarargs
+	public static <E> List<E> combineLists(List<E> ...lists) {
```

Comment:
I'm a bit surprised to see a function with a for-loop instead of something using the streams API, like `lists.stream().flatMap(l -> l.stream()).collect(Collectors.toList());` ;)

...not that I'm suggesting it needs to be changed (this probably performs better and is easier to understand). ;)

Reply:
Yeah, in this case, a for-loop seemed clearer to me and also since I was filtering out any `null` list values.

_"I don't always use functional programming, but when I do I choose streams..." (The most interesting programmer in the world)_